en/docs/deploy-and-publish/publish-on-dev-portal/third-party-api-support.md

```diff
@@ -9,17 +9,18 @@ In previous versions of API Manager, third-party APIs were available when import
 
 1. A regular API can be converted to a third-party API from the **Basic Info** section in the API Publisher portal.
 
-!!! note
-    An API cannot be converted to a third-party API when there are active deployments.
-
-[![Convert to third-party API]({{base_path}}/assets/img/develop/convert-to-third-party.png)]({{base_path}}/assets/img/develop/convert-to-third-party.png)
+    !!! note
+        An API cannot be converted to a third-party API when there are active deployments.
 
-2. In addition to the streaming APIs supported in API Manager (WebSocket, SSE and WebSub), the **Other** option provided when importing AsyncAPI definitions can be used to import a third-party API.
+    [![Convert to third-party API]({{base_path}}/assets/img/develop/convert-to-third-party.png)]({{base_path}}/assets/img/develop/convert-to-third-party.png)
 
-!!! note
-    The APIs created using the ‘Other’ option cannot be converted to a regular API. It can be only used as a third-party API.
+2. In addition to the streaming APIs supported in API Manager (WebSocket, SSE, AsyncAPI, and WebSub) the **Other** option provided when importing API definitions can be used to import a third-party API.
```

Comment:
it is not correct to say `AsyncAPI` here. `AsyncAPI` is merely the API definition for streaming APIs and not a separate API type.

Reply:
Can you please add a screenshot of your output for this page.